karton/classifier/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "1.0.0"
+__version__ = "1.0.1"
```

Comment:
There are already new features on the branch, so we should bump the minor.
```suggestion
__version__ = "1.1.0"
```